src/client/styles/components/_forms.v2.scss

```diff
@@ -0,0 +1,38 @@
+.nypl-fieldset {
```

Comment:
@ricardoom the scss is not complete but it's a step closer to the designs. Let me know if you want anything updated. Example: http://dev-discovery.nypl.org/hold/request/b16069353-i15481004/edd